cidata.py

```diff
@@ -0,0 +1,431 @@
+"""
+Primitive to Build and Delete directories on PodNet HA
+"""
+
+# stdlib
+import json
+import ipaddress
+from pathlib import Path
+from typing import Tuple
+# lib
+from cloudcix.rcc import comms_ssh, CouldNotConnectException
+# local
+
+
+__all__ = [
+    'build',
+]
+
+SUCCESS_CODE = 0
+
+
+def build(
+        domain_path: str,
+        domain: str,
+        interfaces: dict,
+        config_file=None,
+        userdata=""
+) -> Tuple[bool, str]:
+    """
+    description:
+        Creates cloud-init user data and meta data files for a virtual machine on PodNet HA.
+
+    parameters:
+        domain_path:
+            description: path to the virtual machine's cloud-init directory.
```

Comment:
```python
"""
description: |
    path to the virtual machine's cloud-init directory.
    This must be the full path, up to and including the
    metaddata version component.
"""
```

The styling pattern for a multiline value is to add a pipeline `|` after the key and then the value is indented on the next lines under the key

Reply:
Fixed.